composer.json

```diff
@@ -39,14 +39,14 @@
         "composer-runtime-api": "^2.0",
         "dragonmantank/cron-expression": "^3.3.2",
         "laravel/serializable-closure": "^1.2.2",
-        "symfony/config": "^5.4.9 || ^6.0.9",
-        "symfony/console": "^5.4.9 || ^6.0.9",
-        "symfony/dependency-injection": "^5.4.9 || ^6.0.9",
-        "symfony/filesystem": "^5.4.9 || ^6.0.9",
-        "symfony/lock": "^5.4.9 || ^6.0.9",
-        "symfony/mailer": "^5.4.9 || ^6.0.9",
-        "symfony/process": "^5.4.9 || ^6.0.9",
-        "symfony/yaml": "^5.4.9 || ^6.0.9"
+        "symfony/config": "^5.4.9 || ^6.0.9 || ^6.1.0 || ^6.2.0",
```

Comment:
There is no need to add `^6.1.0 || ^6.2.0`, `^6.0.9` is enough.

Reply:
Done.